schema/workflow/unit/io/download_from_object_storage.json

```diff
@@ -0,0 +1,62 @@
+{
+    "schemaId": "workflow-unit-download-from-object-storage",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "download-from-object-storage io schema",
+    "type": "object",
+    "allof": [
+        {
+            "$ref": "../../unit.json"
+        }
+    ],
+    "properties": {
+        "filename": {
+            "description": "name given to the file when it is downloaded",
+            "type": "string"
+        },
+        "objectData": {
+            "description": "information describing the file and where it was run",
+            "type": "object",
+            "properties": {
+                "CONTAINER": {
+                    "description": "Container that ran the job resulting in the file",
```

Comment:
I think this is misunderstanding what a container is.  This is container in the sense of "object store", like an aws s3 bucket: https://aws.amazon.com/s3/.  Not a container in the sense of a containerized process, as in docker.

Reply:
Thanks for pointing that out. Fixed the description